test/integration/jupyter_notebooks_test.py

```diff
@@ -0,0 +1,21 @@
+"""Test that jupyter notebooks can be run."""
+from pathlib import Path
+
+import nbformat
+import pytest
+from nbconvert.preprocessors import ExecutePreprocessor
+
+
+@pytest.mark.parametrize(
+    "notebook",
+    [
+        "notebooks/test_something.ipynb",
```

Comment:
You probably already know this, but this is currently causing my integration tests to fail locally, since this notebook doesn't exist.